### PR TITLE
Cldc 477/reason for leaving because of the benefit cap

### PR DESCRIPTION
**Pull request by MatthewPhelanMadeTech**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -131,6 +131,7 @@
     t.boolean "reasonable_preference_reason_medical_grounds"
     t.boolean "reasonable_preference_reason_avoid_hardship"
     t.boolean "reasonable_preference_reason_do_not_know"
+    t.integer "property_number_of_times_relet"
```
Comment (baarkerlounger) [inline db/schema.rb]: Probably shouldn't be committing schema changes without a migration?

File: app/models/case_log.rb [hunk 2]
```diff
@@ -38,6 +38,12 @@ def validate_other_reason_for_leaving_last_settled_home(record)
       record.errors.add :other_reason_for_leaving_last_settled_home, "The other reason must not be provided if the reason for leaving settled home was not other"
     end
   end
+  
+  def validate_reason_for_leaving_last_settled_home(record)
```
Comment (baarkerlounger) [inline app/models/case_log.rb:42]: Can we add a spec for this?

Comment (MatthewPhelanMadeTech) [reply app/models/case_log.rb:42]: We absolutely can, knew I was forgetting something

File: db/schema.rb (hunk 1 above)
Comment (MatthewPhelanMadeTech) [reply db/schema.rb]: very good point - I assume this is why the tests failed - will fix now

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**